### ISSUE TITLE
image size doubled in 2021.01.11

### ISSUE THREAD

**Issue by willkg** [mozilla-services/minidump-stackwalk#14]
The size of the image went from 244mb to 502mb in 2021.01.11. We should look at reducing it again. Larger image sizes increase the time for Socorro builds.

**Comment by gabrielesvelto** [comment]
Was this a result of the PGO stuff? Maybe we're including things in the final image that shouldn't be there.

**Comment by willkg** [comment]
Yeah, I think I wrote this up when we redid the PGO stuff. I think it's likely that the image includes things it doesn't need like the profiles. I wrote up the issue, but didn't spend any time looking into it.

As a side note, this isn't high on my priorities list. While I'm sure it affects Socorro build times, it's probably not noticeable especially with the noise of everything else.
